apps/notifications/src/panel/templates/status-bar.jsx

```diff
@@ -34,14 +34,14 @@ export class StatusBar extends React.Component {
 	 * `show()` function.
 	 */
 	UNSAFE_componentWillReceiveProps( nextProps ) {
-		if ( '' == nextProps.statusMessage ) return;
+		if ( '' === nextProps.statusMessage ) return;
 
-		if ( nextProps.statusMessage == this.props.statusMessage ) return;
+		if ( nextProps.statusMessage === this.props.statusMessage ) return;
 
-		var component = this;
+		const component = this;
 
 		/* We only want this to appear for a bit, then disappear */
-		var timeout = window.setTimeout(
+		const timeout = window.setTimeout(
 			function () {
 				component.disappear();
 			},
```

Comment:
Can be replaced with `() => { this.disappear(); }`, with the desired `this` binding by default.